pkg/jsonnet/jpath/dirs.go

```diff
@@ -7,6 +7,20 @@ import (
 	"runtime"
 )
 
+func Dirs(path string) (string, string, error) {
```

Comment:
Is not obvious to me by simply looking at the signature what the returned values are, I need to read the function to understand what is being returned. I'm not a big fan of named returns, but I think that here they could help. Alternative a comment could suffice.